public/css/dashboard.css

```diff
@@ -683,4 +732,8 @@ body[data-modal]:after {
     /*color: #92929b;*/
 }
 
+.footer {
```

Comment:
What is this for? If the footer needs to be hidden, why not make its rendering optional in the widget (just like the foldable feature).

Reply:
There is two footer, one in the menu, and another one just below. 
I'm not sure what the footer below is made for, and when / if we can remove it

Reply:
Can you show me a screenshot?

Reply:
![footer](https://user-images.githubusercontent.com/2345926/71979401-e5427680-321d-11ea-988b-ae7eea7ce477.png)

Reply:
Well yeah, but maybe it shouldn't be rendered in the first place; this looks like a bug to me ;-)
It definitely is. I'll fix the bug instead of working around it in css :)

Reply:
it is now fixed in master, you should be able to pull that change into your branch.

Reply:
Ok thanks, it works now. I removed the fix from css